docs/resources/swr_repository_sharing.md

```diff
@@ -0,0 +1,71 @@
+---
+subcategory: "Software Repository for Container (SWR)"
+---
+
+# huaweicloud_swr_repository_sharing
+
+Manages a SWR repository sharing resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "organization_name" {} 
+variable "repository_name" {}
+variable "access_domain" {}
+
+resource "huaweicloud_swr_repository_sharing" "test" {
+  organization  = var.organization_name
+  repository    = var.repository_name
+  access_domain = var.access_domain
+  permission    = "read"
+  deadline      = "forever"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource. If omitted, the
+  provider-level region will be used. Changing this creates a new resource.
+
+* `organization` - (Required, String, ForceNew) Specifies the name of the organization (namespace) the repository belongs.
+  Changing this creates a new resource.
+
+* `repository` - (Required, String, ForceNew) Specifies the name of the repository to be shared.
+  Changing this creates a new resource.
+
+* `access_domain` - (Required, String, ForceNew) Specifies the name of the domain for repository sharing.
```

Comment:
1. suggest to use `sharing_account` and update the description
2. can we support a accounts list?

Reply:
1. updated to sharing_account
2. accounts list is not to easy to support and there is no need for it